ext/node/ops/vm_internal.rs

```diff
@@ -177,17 +186,25 @@ impl ContextifyContext {
 
 pub const VM_CONTEXT_INDEX: usize = 0;
 
-fn create_v8_context<'a>(
-  scope: &mut v8::HandleScope<'a>,
+pub fn create_v8_context<'a>(
+  scope: &mut v8::HandleScope<'a, ()>,
   object_template: v8::Local<v8::ObjectTemplate>,
-  snapshot_data: Option<&'static [u8]>,
+  use_snapshot: bool,
```

Comment:
Consider using an enum instead, it's really not obvious what these bools mean at the call site